charts/metrics-server/README.md

```diff
@@ -33,9 +33,13 @@ The following table lists the configurable parameters of the _Metrics Server_ ch
 | `serviceAccount.name`                | Service account to be used. If not set and `serviceAccount.create` is `true`, a name is generated using the full name template.                                                                                                                                  | `nil`                                      |
+| `serviceAccount.secrets`             | The list of secrets mountable by this service account. See https://kubernetes.io/docs/reference/labels-annotations-taints/#enforce-mountable-secrets                                                                                                             | `[]`                                            |
 | `rbac.create`                        | If `true`, create the RBAC resources.                                                                                                                                                                                                                            | `true`                                     |
 | `rbac.pspEnabled`                    | If `true`, create a pod security policy resource.                                                                                                                                                                                                                | `false`                                    |
 | `apiService.create`                  | If `true`, create the `v1beta1.metrics.k8s.io` API service. You typically want this enabled! If you disable API service creation you have to manage it outside of this chart for e.g horizontal pod autoscaling to work with this release.                       | `true`                                     |
+| `apiService.annotations`             | Annotations to add to the API service                                                                                                                                                                                                                            | `{}`                                            |
+| `apiService.insecureSkipTLSVerify`   | Specifies whether to skip TLS verification                                                                                                                                                                                                                       | `true`                                          |
+| `apiService.caBundle`                | The PEM encoded CA bundle for TLS verification                                                                                                                                                                                                                   | `""`                                            |
```

Comment:
> @chotiwat please update the docs and save the changes without formatting the whole file.

Done. The new values are formatted to align with the longest line, which is the `image.repository` line, i.e. the next autoformat won't include these new lines. I assume you will format the whole table in another PR or commit, right?

Reply:
I don't think the table has been formatted since I first contributed. I'm a big fan of formatting markdown but formatting markdown tables generally hides the changes under a much larger change which makes reviewing much harder.

Reply:
Yeah, makes sense. That's why I'm suggesting another PR just to format the table so subsequent PRs from other contributors unaware of this will have less chance to run into the same issue. Most of the time the added lines won't lengthen the columns so the autoformatting would only affect the new changes (this PR wouldn't have changed other lines if the table were previously autoformatted). I could submit one if you're open to it.